wiki/en/Running-a-Server.md

```diff
@@ -36,86 +37,108 @@ However, various problems can also arise when setting up Servers - especially wh
 
 Consider using a cloud host, not your home internet connection, to get better ping times if you're having problems.
 
-
 ### General notes
 
 * Any Server should have at least 1.6GHz CPU frequency and 1GB RAM
 * Running a Server may require you to adjust any firewalls running on or outside of your machine or cloud host.
-* Running an Unregistered Server at home will require you to [port forward](#port-forwarding) on your router. When running a Registered Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
+* Running an [Unregistered Server](Unregistered-Servers) at home will require you to port forward on your router. This should not be necessary when running a Registered Server, in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
```

Comment:
```suggestion
* Running an [Unregistered Server](Unregistered-Servers) at home will require you to set up port forwarding on your router. This should not be necessary when running a Registered Server in most cases. However, some home networks can require port forwarding for a Registered Server.
```

How common is it for a registered server on a home network to require port forwarding?

Reply:
Not very common, but we have had reports of people solving problems on Registered servers by port forwarding.